### DISCUSSION TITLE
wlr-layer-shell-unstable support?

### DISCUSSION THREAD

**Discussion by trolljoe** [YaLTeR/niri#679] [Q&A]
i'd like to use [wlgreet](https://git.sr.ht/~kennylevinsen/wlgreet) as my greeter but i have no clue if niri supports this protocol. i know sway does, but i dont really wanna risk bricking greetd

**Comment by YaLTeR** [comment]
Niri supports layer shell, but you still need a backup plan if things get bricked. (Like switching to a different tty and fixing stuff from there)
